AtspmApi/Web.config

```diff
@@ -18,15 +18,17 @@
     <!-- 30 days: 30*24 hours-->
     <add key="NumberRecordsThreshold" value="300000" />
     <add key="ApiRole" value="55de4dec-ef4b-4741-b82d-8c4c4432278a" />
-    <add key="LocalArchiveDirectory" value="C:\Users\carter.liu\Documents\Testing" />
+    <add key="LocalArchiveDirectory" value="C:\ATSPMStorage" />
     <add key="StorageLocation" value="0" />
     <add key="FILE_PATH_PREFIX" value="date" />
     <!--with 500,000 rows, ApproachCycleAgg is 27 MB, CtrlerEventLogs 18 MB, SppedEvents 20 MB, others under 20 MB-->
   </appSettings>
   <connectionStrings>
     <add name="Models.AtspmApi" connectionString="data source=(localdb)\localhost;initial catalog=MOE;Persist Security Info=False;User ID=atspm;Password=password;MultipleActiveResultSets=True;App=EntityFramework;Connection Timeout=30" providerName="System.Data.SqlClient" />
-    <add name="AuthContext" connectionString="data source=(localdb)\localhost;initial catalog=MOE;Persist Security Info=False;User ID=atspm;Password=password;MultipleActiveResultSets=True;App=EntityFramework;Connection Timeout=30" providerName="System.Data.SqlClient" />
+    <add name="SPM" connectionString="data source=(localdb)\localhost;initial catalog=MOE;Persist Security Info=False;User ID=atspm;Password=password;MultipleActiveResultSets=True;App=EntityFramework;Connection Timeout=30" providerName="System.Data.SqlClient" />
     <add name="ADConnectionString" connectionString="LDAP://itwcap1" />
+    <add name="Models.AtspmApi" connectionString="data source=(localdb)\localhost;initial catalog=MOE;Persist Security Info=False;User ID=atspm;Password=password;MultipleActiveResultSets=True;App=EntityFramework;Connection Timeout=30" providerName="System.Data.SqlClient" />

```

Comment:
Do you guys see an extra Models.AtspmApi connection string here? I do not see it locally on my machine when I pushed the code.

Reply:
Yes looks like AuthContext was replaced with an extra AtspmApi